docs/pages/management/export-audit-events/splunk.mdx

```diff
@@ -0,0 +1,783 @@
+```
+
+Edit `out.yaml`, adding `teleport-event-handler-impersonator` to the list of
+existing roles.
+
+Apply your changes:
+
+```code
+$ tctl create -f out.yaml
+```
+</TabItem>
+<TabItem label="GitHub">
+
+Retrieve your `github`  configuration resource:
+
+```code
+$ tctl get github/github > github.yaml
+```
+
+Edit `github.yaml`, adding `teleport-event-handler-impersonator` to the
+`teams_to_roles` section. The team you will map to this role will depend on how
+you have designed your organization's RBAC, but it should be the smallest team
+possible within your organization. This team must also include your user.
+
+Here is an example:
+
+```diff
+  teams_to_roles:
+    - organization: octocats 
+      team: admins 
+      roles:
+        - access
++       - teleport-event-handler-impersonator
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f github.yaml
+```
+</TabItem>
+<TabItem label="SAML">
+
+Retrieve your `saml`  configuration resource:
+
+```code
+$ tctl get saml/mysaml > saml.yaml
+```
+
+Edit `saml.yaml`, adding `teleport-event-handler-impersonator` to the
+`attributes_to_roles` section. The attribute you will map to this role will
+depend on how you have designed your organization's RBAC, but it should be the
+smallest group possible within your organization. This group must also include
+your user.
+
+Here is an example:
+
+```diff
+  attributes_to_roles:
+    - name: "groups" 
+      value: "my-group" 
+      roles:
+        - access
++       - teleport-event-handler-impersonator
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f saml.yaml
+```
+</TabItem>
+<TabItem label="OIDC">
+
+Retrieve your `oidc`  configuration resource:
+
+```code
+$ tctl get oidc/myoidc > oidc.yaml
+```
+
+Edit `oidc.yaml`, adding `teleport-event-handler-impersonator` to the
+`claims_to_roles` section. The claim you will map to this role will depend on
+how you have designed your organization's RBAC, but it should be the smallest
+group possible within your organization. This group must also include your
+user.
+
+Here is an example:
+
+```diff
+  claims_to_roles:
+    - name: "groups" 
+      value: "my-group" 
+      roles:
+        - access
++       - teleport-event-handler-impersonator
+```
+
+Apply your changes:
+
+```code
+$ tctl create -f saml.yaml
+```
+</TabItem>
+</Tabs>
+
+Log out of your Teleport cluster and log in again to assume the new role.
+
+### Export the access plugin identity
+
+Like all Teleport users, `teleport-event-handler` needs signed credentials in
+order to connect to your Teleport cluster. You will use the `tctl auth sign`
+command to request these credentials for the plugin. 
+
+<ScopedBlock scope={["enterprise", "oss"]}>
+
+The format of the credentials depends on whether you have set up your network to
+give the plugin direct access to the Teleport Auth Service, or if all Teleport
+clients and services connect to the Teleport Proxy Service instead.
```

Comment:
Since connecting through the proxy is the preferred/suggested configuration, I suggest restructuring this phrasing so that it's seen as the primary option. For example:

```suggestion
The format of the credentials depends on whether you have set up your Teleport
cluster such that all clients and services connect to the Teleport Proxy service,
or to give the plugin direct access to the Teleport Auth Service instead.
```

I'm sure ^ could use some tweaking though.